src/SFA.DAS.AssessorService.Application.Api/Validators/ExternalApi/Certificates/CreateBatchCertificateRequestValidator.cs

```diff
@@ -27,6 +27,7 @@ public class CreateBatchCertificateRequestValidator : AbstractValidator<BatchCer
                 {
                     var existingCertificate = await certificateRepository.GetCertificate(m.Uln, m.StandardCode);
                     var sumbittingEpao = await organisationQueryRepository.GetByUkPrn(m.UkPrn);
+                    var learnerDetails = await ilrRepository.Get(m.Uln, m.StandardId.GetValueOrDefault());
```

Comment:
If there is a possibility that the GetValueOrDefault() would return default null? then it is also possible the call to get the learner details would get nothing and you should handle that gracefully below